libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1179,12 +1126,87 @@ export class RushConfiguration {
    * @deprecated Use `getCommittedShrinkwrapFilename` instead, which gets the correct common
```

Comment:
The `get committedShrinkwrapFilename(): string` getter has been marked as `@deprecated` for a long time.  I think we're safe to remove it.

For the `getCommittedShrinkwrapFilename()` API, I think we can simply replace `variant` with `subspaceName` under the premise that nobody is using Installation Variants, so we can safely remove that API (in a PR immediately following this one).

Reply:
Removed and changed